### PR TITLE
Feature/creating order details

### PR DESCRIPTION
**Pull request by Ramchy-J**
-Creating orders details
-Updating readme
-Creating a list in orders, referenced to OrderDetail

### CODE REVIEW

File: entities/OrderDetails.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+
+public class OrderDetails extends Base {
```
Comment (reynoldmorel) [inline entities/OrderDetails.java]: Rename to `OrderDetail`?

File: entities/OrderDetails.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    //Attributes
+    private Integer orderID;
```
Comment (reynoldmorel) [inline entities/OrderDetails.java]: Instead refer to the whole `Order` entity

File: entities/OrderDetails.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    private Integer productID;
```
Comment (reynoldmorel) [inline entities/OrderDetails.java]: You can refer to the whole `Product` entity

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: entities/OrderDetails.java (hunk 1 above)
Comment (Ramchy-J) [reply entities/OrderDetails.java]: done

File: entities/OrderDetails.java (hunk 3 above)
Comment (Ramchy-J) [reply entities/OrderDetails.java]: done

File: entities/OrderDetails.java (hunk 2 above)
Comment (Ramchy-J) [reply entities/OrderDetails.java]: done

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**